module_3/gift_certificates_system/model/src/main/java/com/epam/esm/entity/UserPrice.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.esm.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.math.BigDecimal;
+
+/**
+ * Contain user with the highest cost of all orders and sum of all orders for this user.
+ * @see com.epam.esm.entity.MostWidelyUsedTag
+ */
+@AllArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
+public class UserPrice {
```

Comment:
It's not an entity, why is it in the "entity" package?

Reply:
Thanks, removed from entity to `pojo` package.